src/containers/LoginPage/loginForm.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { useFormik } from "formik";
+
+import { useDispatch } from "react-redux";
+import loginSchema from "./validations";
+import { login } from "../../store/auth/actions";
+
+const LoginForm = () => {
+  const dispatch = useDispatch();
+  const formik = useFormik({
+    initialValues: {
+      email: "",
+      password: "",
+    },
+    validationSchema: loginSchema,
+    onSubmit: (values) => {
+      dispatch(login(values));
+    },
+  });
+  return (
+    <div className="container w-50 border p-3">
+      <form onSubmit={formik.handleSubmit}>
+        <div className="form-group">
+          <label htmlFor="email">Email:</label>
+          <input
+            type="email"
+            className="form-control"
+            id="email"
+            onChange={formik.handleChange}
+            value={formik.values.email}
+          ></input>
+          {formik.errors.email ? (
```

Comment:
I would create separate function, something like:

```
const renderFormikError = (field) => {
    if (!field) return null;
    
    return (
         <small className="form-text text-muted">
             {formik.errors.email}
         </small>
    );
}
```

You can use this one for multiple error renderings.